src/scss/modifiers/_inverse.scss

```diff
@@ -29,5 +28,75 @@
 				border-color: _oFormsGet('toggle-inverse-selected');
 			}
 		}
+
+		.o-forms-input--invalid {
+			*:invalid:not(:disabled) {
+				color: _oFormsGet('invalid-base-color-inverse');
+				border-color: _oFormsGet('invalid-base-border-inverse');
+
+				+ .o-forms-input__label:before,
+				+ .o-forms-input__label:after {
+					border-color: _oFormsGet('invalid-base-border-inverse');
+				}
+			}
+
+			.o-forms-input__error {
+				color: _oFormsGet('invalid-base-inverse');
+			}
+		}
+
+		.o-forms-input--radio-box {
```

Comment:
It would be great to output these styles conditionally if the radio-box and invalid variant have been added. So if a user is using inverse form inputs but not the radio box they can keep their bundle size small (same for the error summary)

Reply:
Done in [`a986471` (#376)](https://github.com/Financial-Times/o-forms/pull/376/commits/a986471e00aef9ceaff61c43bd498b8ec3404479)